Week 8/RecipeApp/app/src/main/res/layout/fragment_recipe_detail.xml

```diff
@@ -0,0 +1,152 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Your layouts look quite nice and display the information, I noticed some html tags in the data and I don't know if something like this would help: https://stackoverflow.com/questions/2116162/how-to-display-html-in-textview